src/test/util/txmempool.cpp

```diff
@@ -71,14 +71,14 @@ std::optional<std::string> CheckPackageMempoolAcceptResult(const Package& txns,
 
         // Each subpackage is allowed MAX_REPLACEMENT_CANDIDATES replacements (only checking individually here)
         if (atmp_result.m_replaced_transactions.size() > MAX_REPLACEMENT_CANDIDATES) {
-            return strprintf("tx %s result replaced too many transactions",
+            return strprintf("tx %s would replace too many transactions",
                                 wtxid.ToString());
         }
 
         // Replacements can't happen for subpackages larger than 2
         if (!atmp_result.m_replaced_transactions.empty() &&
             atmp_result.m_wtxids_fee_calculations.has_value() && atmp_result.m_wtxids_fee_calculations.value().size() > 2) {
-             return strprintf("tx %s was part of a too-large package RBF subpackage",
+             return strprintf("RBF subpackage with tx %s was too large",
```

Comment:
I think adcfd69e443475f941c0b0f9d1a0246801ab5dad can be dropped.

See https://github.com/bitcoin/bitcoin/pull/28984#discussion_r1665534798. The error strings are correct to be in past tense instead of conditional.

Reply:
removed